pkg/constants/constants.go

```diff
@@ -194,22 +194,30 @@ const (
 var ResourcesToBlock = map[string]bool{
 	// Kubernetes with vSphere Supervisor Cluster resources
 	"agentinstalls.installers.tmc.cloud.vmware.com":      true,
+	"availabilityzones.topology.tanzu.vmware.com":        true,
 	"aviloadbalancerconfigs.netoperator.vmware.com":      true,
 	"blockaffinities.crd.projectcalico.org":              true,
+	"ccsplugins.appplatform.wcp.vmware.com":              true,
 	"certificaterequests.cert-manager.io":                true,
 	"certificates.cert-manager.io":                       true,
 	"challenges.acme.cert-manager.io":                    true,
+	"clusterclasses.cluster.x-k8s.io":                    true,
 	"clusterissuers.cert-manager.io":                     true,
+	"clusternetworkinfos.nsx.vmware.com":                 true,
 	"clusterresourcesetbindings.addons.cluster.x-k8s.io": true,
 	"clusterresourcesets.addons.cluster.x-k8s.io":        true,
 	"clusters.cluster.x-k8s.io":                          true,
+	"cnscsisvfeaturestates.cns.vmware.com":               true,
```

Comment:
Backed up and restored feature state could be different right?.
What is the behavior when we upgrade csi driver before restoring?, for example, backup when csi 2.4 and restore when csi is 2.5, I think this would be a good test to have, please let the QE know such scenarios.

Reply:
dropping